bundles/org.openhab.binding.freeathomesystem/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Any reason you do not want to use the same license that other bindings use?

Reply:
removed